src/vcpkg/commands.dependinfo.cpp

```diff
@@ -163,14 +163,13 @@ namespace vcpkg::Commands::DependInfo
         {
             int empty_node_count = 0;
```

Comment:
```suggestion
            std::size_t empty_node_count = 0;
```

And change

```
s.append(Strings::format("empty [label=\"%d singletons...\"]; }", empty_node_count));
```

below to 

```
s.append(fmt::format("empty [label=\"{} singletons...\"]; }", empty_node_count));
```